nncf/torch/nncf_network.py

```diff
@@ -426,6 +459,8 @@ def __init__(self, module, input_infos: List[ModelInputInfo],
 
         self._compressed_context = TracingContext()
 
+        self._compressed_context.register_global_buffer(STORAGE_DEVICE, device)
```

Comment:
why not to move it to __init__ of `KDLossHandler` by passing required device from NNCFNetwork?